src/libs/sfdk/vboxvirtualmachine.cpp

```diff
@@ -201,8 +201,16 @@ CommandQueue *commandQueue()
  * \class VBoxVirtualMachine
  */
 
+static VirtualMachine::Features staticFeatures()
+{
+    return VirtualMachine::LimitMemorySize | VirtualMachine::LimitCpuCount
+        | VirtualMachine::GrowStorageSize | VirtualMachine::ShrinkStorageSize
+        | VirtualMachine::OptionalHeadless | VirtualMachine::Snapshots;
+}
+
```

Comment:
Ah, I meant static member function, similar to `staticType` and `staticDisplayType`, ordered right after those two.